### PR TITLE
Document used FDB ports

### PR DESCRIPTION
**Pull request by johscheuer**
# Description

Fixes: https://github.com/FoundationDB/fdb-kubernetes-operator/issues/1679

## Type of change

*Please select one of the options below.*

- Documentation

## Discussion

-

## Testing

N/A

## Documentation

Added.

## Follow-up

-

### CODE REVIEW

File: docs/manual/warnings.md [hunk 1]
```diff
@@ -41,6 +41,12 @@ You should be careful when changing images, environment variables, commands, or
 
 Other than the above, you can make any modifications to the pod definition you need to suit your requirements.
 
+## Limitations on FDB Port Customization
```
Comment (sbodagala) [inline docs/manual/warnings.md:44]: How is "processNumber" going to be decided? Is the port going to be assigned to a process (or the additional processes) in a deterministic way?

Comment (johscheuer) [reply docs/manual/warnings.md:44]: The process number is deterministic and is only used if more than one process is running per Pod e.g. when `storageServersPerPod` is set. The first process has the process number 1 and then the next processes will have the incremented process number e.g. 2, 3, 4 .., depending on how many processes should be running per Pod.

**Review by AnnigeriShambu — APPROVED** [review]

File: docs/manual/warnings.md (hunk 1 above)
Comment (sbodagala) [reply docs/manual/warnings.md:44]: So the process number gets assigned/decided based on the order that the processes start?

**Review by sbodagala — APPROVED** [review]

File: docs/manual/warnings.md (hunk 1 above)
Comment (johscheuer) [reply docs/manual/warnings.md:44]: Not really, the operator configures this in the fdbmonitor config.

**Pull request merged**